CMakeLists.txt

```diff
@@ -342,5 +342,31 @@ if(UNIX AND NOT APPLE)
   endif(FREECIV_ENABLE_RULEDIT)
 endif(UNIX AND NOT APPLE)
 
+# We grab the Libertine Font package online for non-Unix (e.g. Win32, MSYS2, and macOS)
+#  For Linux, we add a dependency to the package.
+if(NOT UNIX)
```

Comment:
`UNIX` is true on Mac as well. I suggest making this a configure-time option.
Also, this should only be downloaded when we build the client (thinking of the LT server where we have a restrictive proxy).

Reply:
That is the intention (to get picked up on all but Linux/Unix). I'll add another condition so it only runs if you are building the client.

Reply:
What if I'm on Slitaz and I don't have the `ttf-libertine` package though? I'm for downloading it by default and allowing distributors to use the system version

Reply:
LoL, now that is an edge case since we don't technically support that distro. I'll change it.